models/user.go

```diff
@@ -65,6 +72,17 @@ func (d *UserData) Value() interface{} {
 		return d.NumericValue
 	case BOOL:
 		return d.BoolValue
+	case MAP:
+		data := &map[string]interface{}{}
+		json.Unmarshal([]byte(d.SerializedValue), data)
+		return data
+	case ARRAY:
```

Comment:
I would *love* to find a better way to persist map's and arrays instead of just serializing it to a column.

Any ideas? Especially, I would like to end up with a user record like this:

```
{
   "id": 1234,
   "app_metadata": {
      "roles": ["admin", "cms"]
   }
}
```

And later be able to query the DB for all users with the role "cms"...